### PR TITLE
fix: list filters vertical alignment

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Fix list filters vertical alignment.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen Shot 2021-01-13 at 11 36 42 AM
Screen Shot 2021-01-13 at 4 39 48 PM

### TEST PLAN
1 - Go to main lists (databases, datasets, charts and dashboards)
2 - All filters labels should be vertically aligned with values

@junlincc 

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by willbarrett** [comment]
Hey @michael-s-molina this still looks a pixel low. Better than before though! Happy to merge as-is if there's an issue bumping that 1 more pixel up.

**Comment by michael-s-molina** [comment]
Hey @willbarrett you're absolute right! Thanks for the eagle eyes! I'll fix.

**Comment by rusackas** [comment]
@michael-s-molina I see another route to fixing this that may make sense. Just have to track down the `styled` instances, since Emotion source maps still aren't working :P

In `Base.ts` add `align-items` to `FilterContainer`, i.e.:
```
export const FilterContainer = styled.div`
  display: inline-flex;
  margin-right: 2em;
  font-size: ${({ theme }) => theme.typography.sizes.s}px;
  align-items: center;
`;
```
... and in `FilterTitle` remove the `line-height`, i.e.:
```
  font-weight: bold;
  margin: 0 0.4em 0 0;
`;
```
then you should come out clean without having to nudge any pixels around.

**Comment by michael-s-molina** [comment]
@rusackas Applied your suggestion. Thanks!

**Review by rusackas — APPROVED** [review]

**Comment by rusackas** [comment]
LGTM, but @nytai owns this so I'll let him 👀 & ✅ & 🔀

**Review by nytai — APPROVED** [review]
thanks for the fix!

**Comment by junlincc** [comment]
let's get this one in for rc3. current state looks bad.

**Pull request merged**
